rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -613,24 +620,30 @@ public LRAStatus getStatus(URI uri) throws WebApplicationException {
             }
 
             if (response.getStatus() != OK.getStatusCode()) {
-                LRALogger.i18nLogger.error_invalidStatusCode(coordinatorUrl, response.getStatus(), lraId);
+                String logMsg = LRALogger.i18nLogger.error_invalidStatusCode(coordinatorUrl, response.getStatus(), lraId);
+                LRALogger.logger.error(logMsg);
+                String errorMsg = String.format("%s", logMsg);
```

Comment:
This statement serves no purpose. I notice that you have done this in quite a few places in the PR so I will not mention further examples but it needs fixing.